readme.md

```diff
@@ -116,6 +116,20 @@ normalizeUrl('http://sindresorhus.com/', {removeTrailingSlash: false});
 //=> 'http://sindresorhus.com'
 ```
 
+##### removeDirectoryIndex
+
+Type: `boolean|Array<RegExp|String>`<br>
```

Comment:
You want me to make it `boolean|Array<RegExp|string>` ? Was just following the existing convention in the documentation for `removeQueryParameters`.